examples/src/commonMain/kotlin/ComposeLogo.kt

```diff
@@ -8,7 +8,10 @@ import org.jetbrains.compose.storytale.example.Res
 import org.jetbrains.compose.storytale.example.compose_multiplatform
 
 @Composable
-fun ComposeLogo(size: LogoSize = LogoSize.MEDIUM) {
+fun ComposeLogo(
+  modifier: Modifier = Modifier,
```

Comment:
Hmm. Was this suggested by spotless?

The problem here is that both `modifier` can set sizes and `size` argument too. So they will conflict. And size set in `modifier` would be ignored.

`size: LogoSize = LogoSize.MEDIUM` is a very specific parameter, while Modifier can be very flexible. I'd personally prefer it to be as more specific as possible in this case. 


Do you know which rule exactly forces this?

Reply:
To be honest, it's a difficult question :) 

On one hand, we want to provide a list of "popular" parameters, especially if they need to have default values. It would simplify the usage.

One another hand, when the parameters can conflict with modifiers - it's a problem, because the actual behaviour can be identified only by reading the code or the documentation if it's available.

___
Let's say we wan't the Logo to be only in 3 sizes. But with Modifier it's possible to set any size. 

___

In this case, I guess it's not so important, because it's just an example project - not a library of UI components.